pkg/crd/markers/crd.go

```diff
@@ -179,7 +179,7 @@ func (s PrintColumn) ApplyToCRD(crd *apiext.CustomResourceDefinitionSpec, versio
 
 // Resource defines "+kubebuilder:resource"
 type Resource struct {
-	Path       string
+	Path       string   `marker:",optional"`
```

Comment:
I think we will also have to tweak the apply method for this marker to ignore path if empty. Right ?

Reply:
You are right! Fixed it!